### ISSUE TITLE
Quote styles

### ISSUE THREAD

**Issue by bpdesigns** [slco-2016/beta_report#5]
![screen shot 2016-06-17 at 11 28 16 am](https://cloud.githubusercontent.com/assets/15329140/16165900/9c35dbae-349b-11e6-9999-aa31934f0d81.png)

**Comment by kuanb** [comment]
Luv it

**Comment by kuanb** [comment]
#eb6342
#a83e4e

**Comment by kuanb** [comment]
![image](https://cloud.githubusercontent.com/assets/6053396/16289357/443eab3a-38a9-11e6-9c61-ff44bf53b83a.png)

**Comment by kuanb** [comment]
format

``````

          <div class="quote">
            <div class="phrase">
              "[Texting] would be good for my clients who are homeless or are financially strapped and only use phones for texting."
            </div>
            <div class="attribution">
              - Criminal Justice Services Case Manager
            </div>
          </div>```
``````

**Comment by bpdesigns** [comment]
Looks great!

On Wed, Jun 22, 2016 at 6:50 PM, Kuan Butts notifications@github.com
wrote:

> Closed #5 https://github.com/slco-2016/beta_report/issues/5.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/slco-2016/beta_report/issues/5#event-701315018, or mute
> the thread
> https://github.com/notifications/unsubscribe/AOnndIHYIYkUP5UXQ_Ud_dkV2LASFPWQks5qOeZggaJpZM4I4wwv
> .

## 

*Join me at the Code for America Summit
https://www.codeforamerica.org/summit/?source_codes=staff, November 1 -
3, 2016. *

Ben Peterson // Designer, 2016 _Code for America Fellow
http://www.codeforamerica.org/geeks/fellowship-apply/?source_codes=staff *
// (385) 202-5458 // *@bpdesign https://twitter.com/bpdesign_
